### PR TITLE
Adding Codebook Instructions

### PR DESCRIPTION
**Pull request by prakhar21**
# Status
Please select the status of this PR-

- [x] Ready to review and merge
- [ ] Commits pending
- [ ] On Hold

# Committer Checklist
Adding Codebook Instructions

## Changes Proposed
NA

## Test Results
NA

## Related PRs
NA

## Documentation
NA

# Reviewer Checklist
Please validate and mark the readiness of this PR based on these validations-

## Pre-requisites
- [x] Follows [Coding Style](https://formcept.atlassian.net/wiki/display/DEV/Developer+Handbook#DeveloperHandbook-CodingStyleGuide)
- [x] Follows [Development Workflow](https://formcept.atlassian.net/wiki/display/DEV/Developer+Handbook#DeveloperHandbook-DevelopmentWorkflow) and targets the right branch
- [x] Follows [Commit Message Guidelines](https://formcept.atlassian.net/wiki/display/DEV/Developer+Handbook#DeveloperHandbook-Commit)
- [ ] Has an associated JIRA ID and the same has been mentioned in the PR title

## Change Checklist
- [ ] PR contains only the changes expected in the mentioned JIRA ticket(s)
- [ ] Test Cases have been added
- [ ] Changes have been documented in code and/or Confluence

### CODE REVIEW

File: nbviewer/codebook/codebook_instructions.ipynb [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    "* We have setup our Jupyter Notebook for the below mentioned programming languages:\n",
+    "   1. <b>Python</b>\n",
+    "   2. <b>Scala</b>\n",
+    "   3. <b>Clojure</b>\n",
+    "   4. <b>R</b>\n",
+    "   5. <b>Julia</b>\n",
+    "   6. <b>Octave</b>\n",
+    "   7. <b>Ruby</b>\n",
+    "<br>\n",
+    "* We have bunch of Text Analytics APIs hosted on our server. [Click here](http://lab.formcept.com:7777/nlp/) to know more.\n",
```
Comment (anujsrc) [inline nbviewer/codebook/codebook_instructions.ipynb]: Lets hide this URL and set to something like- localhost:<port> till we hardened the Nginx on our lab machine. Please make this change across.

**Review by anujsrc — CHANGES_REQUESTED** [review]
Change lab.formcept.com to localhost across.

File: nbviewer/codebook/codebook_instructions.ipynb (hunk 1 above)
Comment (prakhar21) [reply nbviewer/codebook/codebook_instructions.ipynb]: @anujsrc Just to localhost or maybe some port also ?

**Review by anujsrc — APPROVED** [review]

**Comment by prakhar21** [comment]
[Click here](https://nbviewer.jupyter.org/github/formcept/whiteboard/blob/master/nbviewer/codebook/codebook_instructions.ipynb) to access the notebook.

**Pull request merged**